packages/vite/src/node/plugins/html.ts

```diff
@@ -221,32 +221,11 @@ export function buildHtmlPlugin(config: ResolvedConfig): Plugin {
   // Same reason with `htmlInlineProxyPlugin`
   isAsyncScriptMap.set(config, new Map())
 
-  const inputFiles = new Set<string>()
-
   return {
     name: 'vite:build-html',
 
-    buildStart({ input }) {
-      isAsyncScriptMap.set(config, new Map())
```

Comment:
Would you check this, `isAsyncScriptMap` should be unrelated to #5342

Reply:
Hmm... the "files changed" on #5342 lists that line as added by the PR, but the actual commits there show it as already present when I started the PR, which is what I remember. I wonder if this was something weird that happened when it merged?

Reply:
Yes, that line should remain. @IanVS would you add the `buildStart` hook with it? GitHub doesn't let me edit deleted lines here

Reply:
In the parent commit, it's outside of the return statement.  https://github.com/vitejs/vite/blob/5d7b4c31b8e44add7c192ae8af4b90b9378ae1fe/packages/vite/src/node/plugins/html.ts#L215, and it's still here in the revert, on line 222 above.  

It looks like the PR added the same `isAsyncScriptMap` line inside of `buildStart`, which is now being removed here.  

So, what change would you like me to make, here?

Reply:
nvm, I read the history wrong, here is the change when we moved out https://github.com/vitejs/vite/pull/6373/files